processor/transform/readme-to-mdx.ts

```diff
@@ -40,6 +40,8 @@ const readmeToMdx = (): Transform => tree => {
 
   visit(tree, 'image', (image, index, parent) => {
     if (!('data' in image)) return;
+
+    if ('url' in image) image.data.hProperties.src = image.url;
```

Comment:
This is the "meat" of the solution. Honestly, I don't love having to do this here, since it feels to me like this ***should*** be happening [in the `imageTransformer`](https://github.com/readmeio/markdown/blob/55dc0c9cffe394ff5e07c28e63fb3c7f94508c9f/processor/transform/images.ts#L25-L29)? am I misunderstanding our Unified order-of-ops here, or something?

Reply:
Yea, it seems like there might be some duplication of code. But this seems for now.